examples/chip-tool/commands/clusters/ModelCommand.h

```diff
@@ -52,6 +52,9 @@ class ModelCommand : public CHIPCommand
                             "Endpoint the command is targeted at.");
             }
         }
+        AddArgument("lit-icd-peer", 0, 1, &mIsPeerLITCLI,
+                    "Whether to treat the peer as a LIT ICD. 0: Always no, 1: Always yes, (not set): Yes if the peer is registered "
```

Comment:
```suggestion
                    "Whether to treat the peer as a LIT ICD. false: Always no, true: Always yes, (not set): Yes if the peer is registered "
```

chip-tool accepts 1 and 0 for booleans for legacy compat reasons, but the documented values for them are `true` and `false`.

Reply:
updated, thanks